docs/UserGuide.adoc

```diff
@@ -151,9 +152,10 @@ Some of the parameters have restrictions and they are stated below.
 * `QUANTITY` has to be a whole number between 0 and 2,000,000 inclusive.
 * `ITEM_NAME` has to be
 * `ITEM_PRICE` has to be
-* `DAY` has to be
-* `MONTH` has to be
-* `YEAR` has to be
+* `YEAR` has to be be within 2000 and the current year.
+* `MONTH` has to be within 1 to 12.
+* `DAY` has to be within 1 to 31 (Special case: 29, 30 and 31 are only vaild if it exists in the month of a specified
```

Comment:
Typo: "valid"; also, phrasing seems a little awkward, consider: (provided it exists in the desired month and year).